slack_scrapper.py

```diff
@@ -1,17 +1,29 @@
 import os
+import requests
+import sys
 from os.path import join, dirname
 
 from dotenv import load_dotenv
 from slackclient import SlackClient
 
 dotenv_path = join(dirname(__file__), '.env')
 load_dotenv(dotenv_path)
-
+ 
 # Get the slack token from the .env file
 token = os.environ.get("SLACK_TOKEN")
 
 sc = SlackClient(token)
 
+#Get all messages in a channel
+def allMessages(channel = 'C5GRK45A9'):
```

Comment:
Hey man, can you try this for the #general channel instead, @andela-rbabalola tried this earlier said he could only get about 127 messages at a time. Can you try it for the #general channel and see if you can get more than 127 messages.